index.html

```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
 
        </div>
    </section>
+
+   <section class="middle-section-work">
+       <div class="middle-section">
+           <h3>My recent work</h3>
+           <hr class="first-line">
+           <div class="stories">
+               <div class="story-left">
+                    <div class="rect"></div>
+                    <div class="desc">
+                        <h4>Multi-Purpose Stories</h4>
+                        <p>A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. has been the industry's 
+                            standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
+                        <div class="skills">
+                            <p>css</p>
+                            <p>html</p>
+                            <p>bootstrap</p>
+                            <p>Ruby</p>
+                        </div>
+                        <button type="button">See Project</button>
```

Comment:
Well-made mocked UI design from your Figma Template ^^! 
_Impressive_

There is another requirement needs to be added 🔽 

![fa8b3dff-60a9-4b8b-9d44-6be30387cd53](https://user-images.githubusercontent.com/24830039/125146504-65ed7600-e126-11eb-9c51-57798c31a0a0.png)